src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessStartInfoTests.cs

```diff
@@ -186,5 +188,59 @@ public void TestUseShellExecuteProperty()
             // Calling the getter
             Assert.False(psi.UseShellExecute, "UseShellExecute=true is not supported on onecore.");
         }
+
+        [Fact]
+        public void TestArgumentsProperty()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo();
+            Assert.Equal(string.Empty, psi.Arguments);
+
+            psi = new ProcessStartInfo("filename", "-arg1 -arg2");
+            Assert.Equal("-arg1 -arg2", psi.Arguments);
+        }
+
+        [Fact]
+        public void TestCreateNoWindowProperty()
+        {
+            Process testProcess = CreateProcessInfinite();
+            try
+            {
+                testProcess.StartInfo.CreateNoWindow = true;
+                testProcess.Start();
+
+                Assert.True(testProcess.StartInfo.CreateNoWindow);
+            }
+            finally
+            {
+                if (!testProcess.HasExited)
+                    testProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+            }
+        }
+
+        [Fact, PlatformSpecific(PlatformID.Windows)]
+        public void TestUserCredentialsPropertiesOnWindows()
+        {
+            // test the defaults here.
+            Assert.Equal(string.Empty, _process.StartInfo.Domain);
+            Assert.Equal(string.Empty, _process.StartInfo.UserName);
+            Assert.Equal(default(SecureString), _process.StartInfo.Password);
+            Assert.False(_process.StartInfo.LoadUserProfile);
```

Comment:
> but those APIs are not available on .NET Core.

Are there Win32 APIs we could P/Invoke to?  I'm not suggesting we do this now, and not for inner loop, but it's something we should have tests for somewhere at some time.